headers.lisp

```diff
@@ -26,6 +26,22 @@
 ;;; NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 ;;; SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+;;; The following code is for treating headers which exist in the keyword package as keywords, otherwise they will be treated as strings. This code will override the chunga code which by default interns every header to be a keyword
+
+(in-package :chunga)
+
+(defvar *intern-unsafely*)
```

Comment:
Initialize this with `nil` to simplify the check in `as-keyword`.

Reply:
just added it